backends/rapidpro/backend.go

```diff
@@ -777,41 +775,33 @@ func (b *backend) Heartbeat() error {
 	redisStats := b.rp.Stats()
 
 	dbWaitDurationInPeriod := dbStats.WaitDuration - b.stats.dbWaitDuration
-	dbWaitCountInPeriod := dbStats.WaitCount - b.stats.dbWaitCount
 	redisWaitDurationInPeriod := redisStats.WaitDuration - b.stats.redisWaitDuration
-	redisWaitCountInPeriod := redisStats.WaitCount - b.stats.redisWaitCount
 
 	b.stats.dbWaitDuration = dbStats.WaitDuration
-	b.stats.dbWaitCount = dbStats.WaitCount
 	b.stats.redisWaitDuration = redisStats.WaitDuration
-	b.stats.redisWaitCount = redisStats.WaitCount
 
 	hostDim := cwatch.Dimension("Host", b.config.InstanceID)
-	appDim := cwatch.Dimension("App", "courier")
```

Comment:
since the namespace is `Temba/Courier` app doesn't make sense as a dimension.. can update mailroom to match this